### PR TITLE
Update content

### PR DESCRIPTION
**Pull request by karlchillmaid**
## What problem does the pull request solve?
This PR updates our PHP client documentation to improve link consistency, the labelling and structure of numbered instructions.

## Checklist

- [x] I’ve used the pull request template
- [ ] I’ve written unit tests for these changes
- [ ] I’ve updated the documentation (in `DOCUMENTATION.md` and `CHANGELOG.md`)
- [ ] I’ve bumped the version number (`const VERSION` in `src/Client.php`)

### CODE REVIEW

File: DOCUMENTATION.md [hunk 1]
```diff
@@ -1045,7 +1058,7 @@ You can also set up [callbacks](#callbacks) for received text messages.
 
 ## Enable received text messages
 
-Contact the GOV.UK Notify team on the [support page](https://www.notifications.service.gov.uk/support) or through the [Slack channel](https://ukgovernmentdigital.slack.com/messages/C0E1ADVPC) to enable receiving text messages for your service.
+Contact the Notify team using the [support page](https://www.notifications.service.gov.uk/support) or [chat to us on Slack](https://ukgovernmentdigital.slack.com/messages/C0E1ADVPC) to request a unique number for text message replies.
```
Comment (idavidmcdonald) [inline DOCUMENTATION.md]: Just checking, we want to drop the `GOV.UK` from `GOV.UK Notify` in this case?

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -361,11 +369,13 @@ If the request is not successful, the client returns an `Alphagov\Notifications\
 
 ### Prerequisites
 
-When your service first signs up to GOV.UK Notify, you’ll start in [trial mode](https://www.notifications.service.gov.uk/features/using-notify#trial-mode). You can only send letters in live mode. You must ask GOV.UK Notify to make your service live.
+When you add a new service it will start in [trial mode](https://www.notifications.service.gov.uk/features/trial-mode). You can only send letters when your service is live.
 
-1. Sign in to [GOV.UK Notify](https://www.notifications.service.gov.uk/).
-1. Select __Using Notify__.
-1. Select __requesting to go live__.
+To send Notify a request to go live:
```
Comment (idavidmcdonald) [inline DOCUMENTATION.md]: Here is another use of Notify without GOV.UK. Just to be clear, I don't have a strong opinion on whether this is best or not but just wanted to point out a potential inconsistency that I'll leave you to make a decision on.

File: DOCUMENTATION.md (hunk 1 above)
Comment (karlchillmaid) [reply DOCUMENTATION.md]: Good spot. Whether or not we include the GOV.UK is about context.

There's a useful rule here: https://github.com/alphagov/notifications-manuals/wiki/A-to-Z-of-style#govuk-notify) but we need to tailor it for documentation.

I don't think it makes a _huge_ difference either way in this case – but adding the GOV.UK is probably the better choice.

This issue may exist in other PRs.

Comment (idavidmcdonald) [reply DOCUMENTATION.md]: Thanks for linking to the style guide to back this up. This issue does indeed exist for other PRs (appears to be in the slack link section and also in `To send Notify a request to go live`). I'll stop pointing it out in PR reviews as it is the same across the clients and leave you to decide if any changes are needed.

File: DOCUMENTATION.md (hunk 2 above)
Comment (karlchillmaid) [reply DOCUMENTATION.md]: The rule we use on the app is:

>Use GOV.UK Notify the first time you mention the product on a page. After that you can call it Notify.

The adaptation of this rule Jon and I discussed for the docs is to use the full name the first time it's mentioned in a new section, then it's at our discretion to drop the 'GOV.UK' if Notify is mentioned several times in context.

This looks like an example of that.

It's probably fine for now, but we may revisit in the future if the rule doesn't work for docs the same way it does for each page of the app.

**Pull request merged**